test/reporter/test-reporter-util.js

```diff
@@ -79,6 +79,7 @@ test('util.sanitizeOutput', function (t) {
   var raw = fs.readFileSync(carriageReturnPath, 'utf-8');
   var expected = fs.readFileSync(carriageReturnExpectedPath, 'utf-8');
   var result = util.sanitizeOutput(raw, '#');
-  t.equals(result, expected, 'there should be a # on every line');
+  result += '\n';
```

Comment:
my text editor added a newline to test/fixtures/CR-raw.txt  which wasn't previously there. Therefore the output needs a newline to match. We are often doing this in the tests when comparing against text fixtures: https://github.com/nodejs/citgm/blob/master/test/reporter/test-reporter-junit.js#L95